http-verbs-common/src/main/scala/uk/gov/hmrc/http/HttpResponse.scala

```diff
@@ -44,6 +46,11 @@ trait HttpResponse {
   def json: JsValue =
     Json.parse(body)
 
+  // If we were using wsResponse, we could have a single body function...
+  //def body[T: BodyReadable]: T = super.body[T]
+  def bodyAsSource: Source[ByteString, _] =
```

Comment:
As well as needing to keep `HttpResponse` to allow us to substitute the WSResponse stream with our audited stream, we need to expose on `HttpResponse`.
It defaults to the strict body, but can be provided by apply function.